server/shd.py

```diff
@@ -0,0 +1,64 @@
+"""
+Starts Gluster Brick(fsd) process
+"""
+import os
+import uuid
+import sys
+import json
+import logging
+import requests
+
+from jinja2 import Template
+
+from kadalulib import logf, send_analytics_tracker
+
+
+VOLFILES_DIR = "/kadalu/volfiles"
+TEMPLATES_DIR = "/kadalu/templates"
+VOLINFO_DIR = "/var/lib/gluster"
+
+
+def generate_shd_volfile(volfile_path, volname):
+    """Generate Client Volfile for Glusterfs Volume"""
+    info_file_path = os.path.join(VOLINFO_DIR, "%s.info" % volname)
+    data = {}
+    with open(info_file_path) as info_file:
+        data = json.load(info_file)
+
+    template_file_path = os.path.join(TEMPLATES_DIR, "Replica3.shd.vol.j2")
```

Comment:
with this feature, should we disable client self-heal altogether? or is it ok to keep it for couple of releases?

Reply:
I am not sure, we can keep it if no perf penalty.

Reply:
There is some perf penalty for sure, I am not aware at present, how much.